samples/highcharts/demo/annotates-tsla-stock-price/demo.js

```diff
@@ -0,0 +1,232 @@
+const commonOptions = {
+    api: {
+        url: 'https://demo-live-data.highcharts.com',
+        access: {
+            url: 'https://demo-live-data.highcharts.com/token/oauth',
+            token: 'token'
+        }
+    }
+};
+
+const TeslaISIN = 'US88160R1014';
+
+// eslint-disable-next-line no-undef
```

Comment:
I think this ESLint disabling is redundant. `HighchartsConnectors` is globally enabled in eslintrc.